js-packages/@prestojs/codegen/tsconfig.lint.json

```diff
@@ -4,7 +4,8 @@
   "compilerOptions": {
     "composite": true,
     "rootDir": "./src",
-    "tsBuildInfoFile": "build/tsconfig.tsbuildinfo"
+    "tsBuildInfoFile": "/tmp/tsc-output/@prestojs/codegen.tsbuildinfo",
+    "outDir": "/tmp/tsc-output/@prestojs/codegen"
```

Comment:
for the linting the files have to be emitted somewhere otherwise has to do full compilation everytime (slow) so workaround is to output them somewhere - so currently hardcoded to /tmp